inc/helpers/namespace.php

```diff
@@ -442,6 +442,7 @@ function get_metakeys() {
 		'pb_book_license' => __( 'License', 'pressbooks-book' ),
 		'pb_primary_subject' => __( 'Primary Subject', 'pressbooks-book' ),
 		'pb_additional_subjects' => __( 'Additional Subject(s)', 'pressbooks-book' ),
+		'pb_institutions' => _n_noop( 'Institution', 'Institutions', 'pressbooks-book' ),
```

Comment:
Will render `Institution` or `Institutions` based on the number of institutions.